tests/k8s/k8s_test_utilities.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import time
+
+logger = logging.getLogger(__name__)
+
+WAIT_FOR_SYNC = 60 # time unit seconds
+
+
+def join_master(duthost, master_vip):
+    """
+    Joins DUT to Kubernetes master
+
+    Args:
+        duthost: DUT host object
+        master_vip: VIP of high availability Kubernetes master
+
+    If join fails, test will fail at the assertion to check_connected
+    """
+    logger.info("Joining DUT to Kubernetes master")
+    duthost.shell('sudo config kube server disable on')
+    duthost.shell('sudo config kube server ip {}'.format(master_vip))
+    duthost.shell('sudo config kube server disable off')
```

Comment:
Use module `ansible/library/shell_cmds.py`?

Reply:
Changed to use `ansible/library/shell_cmds.py`